crates/router/src/core/payments/helpers.rs

```diff
@@ -3889,3 +3892,12 @@ pub fn validate_session_expiry(session_expiry: u32) -> Result<(), errors::ApiErr
         Ok(())
     }
 }
+
+pub fn update_supported_connectors(
```

Comment:
Nit: Is this function really required? Cuz it's too simple/small. If it's only being used at one place, remove it.